### PR TITLE
lite: Add version 1.11

### LINKED ISSUE #4494: [Request] Add lite
[https://github.com/rxi/lite](https://github.com/rxi/lite)

A lightweight text editor written in Lua.

### PR DESCRIPTION
**Pull request by sebmaynard**
- Closes #4494

### CODE REVIEW

File: bucket/lite.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "version": "1.11",
+    "description": " A lightweight text editor written in Lua ",
```
Comment (Ash258) [inline bucket/lite.json:3]:
```suggestion
    "description": "A lightweight text editor",
```

File: bucket/lite.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    "homepage": "https://github.com/rxi/lite",
+    "license": "MIT",
+    "url": "https://github.com/rxi/lite/releases/download/v1.11/lite.zip",
```
Comment (Ash258) [inline bucket/lite.json:6]:
```suggestion
    "architecture": {
        "64bit": {
            "url": "https://github.com/rxi/lite/releases/download/v1.11/lite.zip",
            "hash": "da3701cf6985bc493cb11b37556325551c7ce8606f5502ff3f15199039835eef"
        }
    },
```

File: bucket/lite.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    "bin": "lite.exe",
+    "shortcuts": [
+        [
+            "lite.exe",
+            "lite"
+        ]
+    ],
+    "checkver": "github",
+    "autoupdate": {
+        "url": "https://github.com/rxi/lite/releases/download/v$version/lite.zip"
```
Comment (Ash258) [inline bucket/lite.json:16]:
```suggestion
        "architecture": {
            "64bit": {
                "url": "https://github.com/rxi/lite/releases/download/v$version/lite.zip"
            }
        }
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Comment by ProfElements** [comment]
Sorry to ressurect this old thread. Can I do this since it have been dead for 27 days?

**Comment by sebmaynard** [comment]
@ProfElements of course, sorry life got in my way...

**Comment by ProfElements** [comment]
:+1: Thank you for confirmation

**Comment by pratikpc** [comment]
@sebmaynard could you add Persist support?

**Comment by rashil2000** [comment]
Closing in favour of #7301

**Pull request closed without merging**